production/bin/start.sh

```diff
@@ -17,6 +17,19 @@ function config_replace() {
 	sed -i'.bak' -e ${CMD} ${FILE}
     fi
 }
+function getParsedGithubToken(){
```

Comment:
As per https://mantis.meedan.com/view.php?id=7812, we no longer need `production/bin` since it was used for Linode deployments. So I suggest to not touch that folder for this present ticket.

Reply:
Okay great to know :)